sdk/servicebus/service-bus/src/receivers/sessionReceiver.ts

```diff
@@ -535,7 +535,7 @@ export class SessionReceiverImpl<ReceivedMessageT extends ReceivedMessage | Rece
 
       const receivedMessages = await this._messageSession!.receiveMessages(
         maxMessageCount,
-        options?.maxWaitTimeSeconds
+        options?.maxWaitTimeInMs || Constants.defaultOperationTimeoutInMs
```

Comment:
The doc string needs to be updated as well since it refers to `maxWaitTimeSeconds`.

Reply:
Fixed with https://github.com/Azure/azure-sdk-for-js/pull/8169/commits/be288a9d51f6e065b4183d00de7cd1ebf4586e72